app/views/users/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<section class="d-flex flex-column justify-content-center align-items-center">
+  <h1 class='fw-bold'>Personal Transactions</h1>
+  <div class="d-grid gap-3 col-6 mx-auto">
+    <button class="btn btn-outline-primary" type="button">
+      <%= link_to "Log in", new_user_session_path, data: {turbo: false} %>
+    </button>
+    <button class="btn btn-outline-success" type="button">
+      <%= link_to "Register", new_user_registration_path, data: {turbo: false} %>
+    </button>
+  </div>
+</section>
```

Comment:
- [ ] [Optional][Highly recommended] Please edit your splash screen style and elements placement to match the given design.

| Given Design  | Your Design |
| ------------- | ------------- |
| ![image](https://user-images.githubusercontent.com/68796232/227513638-906c5fa5-a9e6-45c4-990d-a4bd6fcac2ed.png)  | ![image](https://user-images.githubusercontent.com/68796232/227513595-ae261f5d-9c88-4772-83a3-520d5209bbed.png)  |

### ⚠️I marked this as optional because the last reviewer forgot to mention it so please consider doing it⚠️